Practice1/src/css/android.css

```diff
@@ -0,0 +1,73 @@
+.android {
+    display: flex;
+    justify-content: space-around;
+    background-color: var(--blue-light);
```

Comment:
Tab size should be 2 in all files. Please help to update all.

Reply:
Fixed